js/views/itemShortVw.js

```diff
@@ -41,7 +43,23 @@ module.exports = Backbone.View.extend({
       }
 
       this.model.set('ownFollowing', false);
-    });    
+    });
+
+    this.listenTo(window.obEventBus, "blockingUser", function(e){
+      if (e.guid !== this.model.get('guid')) {
+        return;
+      }
+
+      this.model.set('isBlocked', true);
```

Comment:
Just curious, why the return instead of 

if (e.guid == this.model.get('guid')) {
  this.model.set('isBlocked', true);
}